pkg/output/output.go

```diff
@@ -0,0 +1,22 @@
+package output
+
+const (
+	TypeDefault        = ""
```

Comment:
These are the output types supported by `cli-runtime` print flags. Keeping them here so we wouldn't have to pass strings around